### DISCUSSION TITLE
Documentation and samples for SharePointPnPCore2019

### DISCUSSION THREAD

**Discussion by kohsel** [pnp/pnpcore#1149] [Q&A]
Hello.

I have managed to download and create a very basic Hello World project with SharePointPnPCore2019. However I would really like if someone could give me a pointer to the documentation for the library and examples of usage.

Thanks,
Lars Kohsel

**Comment by jansenbe** [comment]
Hi @kohsel ,

PnP Core SDK is an SDK that does not work with SharePoint 2019. Actually all PnP .NET libraries only support SharePoint Online. The old library you're referring to can still be used, but will not evolve nor get fixed. There's no formal doc for that library, but most of the PnP Framework posts on the internet are also relevant for that one.
